client/components/review.js

```diff
@@ -28,13 +29,13 @@ class Review extends React.Component {
       value = options.find(option => !isUndefined(option.value) ? option.value === value : option === value)
     }
 
-    if (value && this.props.type === 'duration') {
+    if (this.props.type === 'duration') {
       return (
         <dl className="inline">
           <dt>Years</dt>
-          <dd>{value.years || 0}</dd>
+          <dd>{(value || {}).years || 5}</dd>
```

Comment:
Does this default to 5 years if it's not set?

Reply:
there's some code in resolver which defaults it to 5 years